docs/src/user_docs/assembly/code_organization.md

```diff
@@ -198,6 +198,47 @@ end
 
 ```
 
+#### Word constants
+
+Along with the regular value constants a word constants could be used. They could be declared as an array of four elements or as a long hex value, and then could be used in the `push` instructions referenced by their name. Notice that a word constant can not be used in a constant expression.
+
+```
+const.SAMPLE_WORD=[1,2,3,4]
+const.SAMPLE_HEX_WORD=0x0200000000000000030000000000000004000000000000000500000000000000
+
+begin
+    push.SAMPLE_WORD       # is equivalent to push.1.2.3.4
+    push.SAMPLE_HEX_WORD.6 # is equivalent to push.2.3.4.5.6
+end
+```
+
+#### Word constant slices
+
+It is possible to get just some part of a word constant using a slice notation. This could be done by using a range in the square brackets right after a constant name. Attempt to get a slice from a constant which doesn't represent a word will return an error. 
```

Comment:
```suggestion
It is possible to get just some part of a word constant using slice notation. This could be done by specifying a range in square brackets right after the constant's name. Attempt to get slices from constants which don't represent words will result in errors. 
```